src/ghdcbot/adapters/storage/sqlite.py

```
@@ -0,0 +1,378 @@
+                SELECT discord_user_id, expires_at
+                FROM identity_links
+                WHERE github_user = ? AND verified = 0
+                """,
+                (github_user,),
+            ).fetchone()
+            if row and row["discord_user_id"] != discord_user_id:
+                existing_expires = row["expires_at"]
+                if existing_expires:
+                    existing_dt = _parse_utc(existing_expires)
+                    if existing_dt > now:
+                        raise ValueError("github_user has an active pending claim by another Discord user")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)